Formula/ftjam.rb

```diff
@@ -18,4 +18,23 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    (testpath/"Jamfile").write <<-EOS.undent
+      Main ftjamtest : ftjamtest.c ;
+    EOS
+
+    (testpath/"ftjamtest.c").write <<-EOS.undent
+      #include <stdio.h>
+
+      int main(void)
+      {
+          printf("FtJam Test\\n");
+          return 0;
+      }
+    EOS
+
+    assert_match /Cc ftjamtest.o/, shell_output(bin/"jam").strip
```

Comment:
It doesn't actually need to be a regex; about to force-push.

Want me to submit a PR for `jam` to change that one too once this is merged, or should we just leave it?